mmv1/templates/terraform/examples/netapp_volume_replication_create.tf.erb

```diff
@@ -0,0 +1,54 @@
+
+data "google_compute_network" "default" {
+  name = "<%= ctx[:vars]['network_name'] %>"
+}
+
+resource "google_netapp_storage_pool" "source_pool" {
+  name          = "<%= ctx[:vars]['source_pool_name'] %>"
+  location      = "us-central1"
+  service_level = "PREMIUM"
+  capacity_gib  = 2048
+  network       = data.google_compute_network.default.id
+}
+
+resource "google_netapp_storage_pool" "destination_pool" {
+  name          = "<%= ctx[:vars]['destination_pool_name'] %>"
+  location      = "us-west2"
+  service_level = "PREMIUM"
+  capacity_gib  = 2048
+  network       = data.google_compute_network.default.id
+}
+
+resource "google_netapp_volume" "source_volume" {
+  location     = google_netapp_storage_pool.source_pool.location
+  name         = "<%= ctx[:vars]['volume_name'] %>"
+  capacity_gib = 100
+  share_name   = "<%= ctx[:vars]['volume_name'] %>"
+  storage_pool = google_netapp_storage_pool.source_pool.name
+  protocols = [
+    "NFSV3"
+  ]
+  deletion_policy = "FORCE"
+}
+
+resource "google_netapp_volume_replication" "<%= ctx[:primary_resource_id] %>" {
+  depends_on           = [google_netapp_volume.source_volume]
+  location             = google_netapp_volume.source_volume.location
+  volume_name          = google_netapp_volume.source_volume.name
+  name                 = "<%= ctx[:vars]['replication_name'] %>"
+  replication_schedule = "EVERY_10_MINUTES"
+  description          = "This is a replication resource"
+  destination_volume_parameters {
+    storage_pool = google_netapp_storage_pool.destination_pool.id
+    volume_id    = "<%= ctx[:vars]['destination_volume'] %>"
+    # Keeping the share_name of source and destination the same
+    # simplifies implementing client failover concepts
+    share_name  = "<%= ctx[:vars]['volume_name'] %>"
+    description = "This is a replicated volume"
+  }
+  # WARNING: Setting delete_destination_volume to true, will delete the
+  # CURRENT destination volume if the replication is deleted. This is great
+  # for testing, but questionable for production!
```

Comment:
```suggestion
  # WARNING: Setting delete_destination_volume to true, will delete the
  # CURRENT destination volume if the replication is deleted. Omit the field 
  # or set delete_destination_volume=false to avoid accidental volume deletion.
```